etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/ruleengine/transformation/custom/UpdatePatientIdentifierListAssigningAuthority.java

```diff
@@ -0,0 +1,30 @@
+package gov.hhs.cdc.trustedintermediary.etor.ruleengine.transformation.custom;
+
+import gov.hhs.cdc.trustedintermediary.etor.ruleengine.FhirResource;
+import gov.hhs.cdc.trustedintermediary.etor.ruleengine.RuleExecutionException;
+import gov.hhs.cdc.trustedintermediary.etor.ruleengine.transformation.CustomFhirTransformation;
+import gov.hhs.cdc.trustedintermediary.external.hapi.HapiHelper;
+import java.util.Map;
+import org.hl7.fhir.r4.model.Bundle;
+
+public class UpdatePatientIdentifierListAssigningAuthority implements CustomFhirTransformation {
+
+    @Override
+    public void transform(FhirResource<?> resource, Map<String, String> args)
+            throws RuleExecutionException {
+
+        if (!(resource.getUnderlyingResource() instanceof Bundle)) {
+            throw new RuleExecutionException("Resource provided is not a Bundle");
+        }
+
+        try {
+            Bundle bundle = (Bundle) resource.getUnderlyingResource();
```

Comment:
This would be the only check that would trigger an exception as the `HapiHelper::updatePatientIdentifierValue` does not throw any exceptions. So, I will add the ability to throw exceptions if the bundle is missing the patient resource or the organization/id can't be found.